.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -72,6 +72,27 @@ on:
         description: The artifact name of the signed provenance. The file must have the .intoto extension. Defaults to <filename>.intoto for single artifact or multiple.intoto.jsonl for multiple artifacts.
         required: false
         type: string
+      upload-assets:
+        description: >
+          If true, provenance is uploaded to a GitHub release for new tags.
```

Comment:
"If true" makes it sound like the default is false. In reality if this is not provided then we will upload to a release by default.